drivers/hmis/app/models/hmis/workflow_definition/node.rb

```diff
@@ -54,6 +54,23 @@ def describe_as_string
       str
     end
 
+    # Returns a string representation of the node for Mermaid diagrams
+    def to_mermaid_node
+      # differentiate shape between start/end events and task nodes
+      if entrypoint? || endpoint?
+        "#{to_mermaid_node_id}((\"#{name}\"))"
+      elsif gateway?
+        "#{to_mermaid_node_id}{\"#{name}\"}"
+      else
+        "#{to_mermaid_node_id}(\"#{name}\")"
+      end
+    end
+
+    # Returns identifier for the node to use in Mermaid diagrams
+    def to_mermaid_node_id
+      name.parameterize(separator: '_')
```

Comment:
should this include the id for uniqueness? 
```suggestion
      "#{name.parameterize(separator: '_')}_#{id}"
```